### PR TITLE
west.yml: Snapshot update of mcumgr from upstream

### PR DESCRIPTION
**Pull request by de-nordic**
This commit updates mcumgr with latest snapshot from the upstream.

Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>

Snapshot update reviewed here: https://github.com/zephyrproject-rtos/mcumgr/pull/28

### CODE REVIEW

**Review by galak — APPROVED** [review]

**Comment by galak** [comment]
Marked DNM while we wait for CI to pass and than merge the module PR side.

**Comment by de-nordic** [comment]
@galak Probably I will not be able to update the SHA today, so could you do that for me when CI passes? Thanks in advance.

**Comment by galak** [comment]
> @galak Probably I will not be able to update the SHA today, so could you do that for me when CI passes? Thanks in advance.

Sure thing, I know its late for you and would be good to get this fixed ASAP.  I'll handle updating the SHA once CI finishes and we merge the module side

**Review by nashif — APPROVED** [review]

**Pull request merged**
